resources/sass/pages/_committee.scss

```diff
@@ -0,0 +1,127 @@
+.ris-committee {
+    .ris-section-wrapper {
+        border: 0;
+    }
+    &__wrapper {
+        display: flex;
+        margin-bottom: 36px;
+        @media only screen and (max-width : 767px) {
+            margin-bottom: $indent-l;
+        }
+    }
+    &-navigation {
+        position: fixed;
+        right: 15px;
+        top: 50%;
+        transform: translateY(-50%);
+        &__button {
+            text-transform: uppercase;
+            padding: 7px 3px;
+            background-color: transparent;
+            border: 0;
+            outline: none;
+            &.bolt {
+                font-weight: bold;
+            }
+        }
+        @media only screen and (max-width : 767px) {
+            display: none;
+        }
+    }
+    &__subtitle {
+        margin-bottom: 10px;
+    }
+    &__search {
+        width: 472px;
+        .ris-search__input {
+            background-color: $c-white;
+            height: 50px;
+            box-shadow: none;
+        }
+    }
+    &__select {
+        @media only screen and (max-width : 767px) {
+            margin-left: $indent-m;
+        }
+    }
+    &__select,
+    &__search {
+        @media only screen and (max-width : 767px) {
+            width: 50%;
+        }
+    }
+    &-main-list {
+        &__heading {
+            text-transform: uppercase;
+        }
+        &__item {
+            &:not(:last-child) {
+                margin-bottom: 52px;
+                @media only screen and (max-width : 767px) {
+                    margin-bottom: $indent-ml;
+                }
+            }
+        }
+    }
+    &-secondary-list {
+        &__item {
+            border-bottom: 1px solid $c-gray7;
+            &-wrapper {
+                color: $c-gray8;
+                font-size: $fs-l;
+                display: flex;
+                align-items: center;
+                @media only screen and (max-width : 767px) {
+                    display: none;
+                }
+            }
+        }
+        &__counter {
+            margin-right: $indent-m;
+            &::before {
+                content: 'X';
```

Comment:
i thought that you have a dedicated scss module for all icons, why do you need to define them here?